### ISSUE TITLE
将intersection组件设为"intersect_method": "raw"，卡在hetero_lr

### ISSUE THREAD

**Issue by jennylove10** [FederatedAI/FATE#1486]
设"intersect_method": ["raw"，用的是样例examples\federatedml-1.x-examples\intersect\test_intersect_job_raw_conf.json中的数据breast_a_intersect/breast_b_intersect。在后面加入hetero_lr组件会报如下错误
![微信图片_20200616145718](https://user-images.githubusercontent.com/62371712/84741468-bb1d8d80-afe1-11ea-8c29-e22c40ac022c.png)
请问是目前raw只支持到intersection这一组件不支持后续加别的组件吗？

**Comment by tanmc123** [comment]
@jennylove10 可以看看intersect组件的输出，看看交集的数据是否正常的？如果还不行的话，麻烦提供一下conf配置

**Comment by jennylove10** [comment]
> @jennylove10 可以看看intersect组件的输出，看看交集的数据是否正常的？如果还不行的话，麻烦提供一下conf配置

@tanmc123 交集的data_output查看了是正常的，conf配置如下：
{
    "initiator": {
        "role": "guest",
        "party_id":9999
    },
    "job_parameters": {
        "work_mode": 1
    },
    "role": {
        "guest": [9999],
        "host": [10000],
        "arbiter": [10000]
    },
    "role_parameters": {
        "guest": {
            "args": {
                "data": {
                    "train_data": [{"name": "breast_b_intersect", "namespace": "hetero"}]
                }
            },
            "dataio_0":{
                "with_label": [true],
                "label_name": ["y"],
                "label_type": ["int"],
                "output_format": ["dense"]
            }
        },
        "host": {
            "args": {
                "data": {
                    "train_data": [{"name": "breast_a_intersect", "namespace": "hetero"}]
                }
            },
             "dataio_0":{
                "with_label": [false],
                "output_format": ["dense"]
            }
        }
    },
    "algorithm_parameters": {
	"intersection_0": {
          "intersect_method": "raw",
          "sync_intersect_ids": true,
          "join_role": "host",
          "with_encode": true,
          "only_output_key": true,
          "encode_params":{
              "encode_method": "sha256",
              "salt": "12345",
              "base64": true
          },
            "hetero_lr_0": {
      "penalty": "L2",
      "optimizer": "nesterov_momentum_sgd",
      "tol": 1e-4,
      "alpha": 0.01,
      "max_iter": 10,
      "early_stop": "weight_diff",
      "batch_size": -1,
      "learning_rate": 0.15,
      "init_param": {
        "init_method": "random_uniform"
      },
      "sqn_param": {
        "update_interval_L": 3,
        "memory_M": 5,
        "sample_size": 5000,
        "random_seed": null
      },
      "cv_param": {
        "n_splits": 5,
        "shuffle": false,
        "random_seed": 103,
        "need_cv": false,
        "evaluate_param": {
          "eval_type": "binary"
        }
      }
    }
  }
}
}

**Comment by tanmc123** [comment]
![image](https://user-images.githubusercontent.com/28244886/84747804-57e42900-afea-11ea-80cd-74210e9bf4fc.png)
把这个only_output_key 设置为False试试？这么设置会只输出可以而没有特征的

**Comment by jennylove10** [comment]
> ![image](https://user-images.githubusercontent.com/28244886/84747804-57e42900-afea-11ea-80cd-74210e9bf4fc.png)
> 把这个only_output_key 设置为False试试？这么设置会只输出key而没有特征的

@tanmc123 改为only_output_key可以跑通啦,想再请教一下专家想把交集中具体传输密文要如何打印出来呐（打印的一直是这种object,有些我查看了源码是私有变量)
![QQ截图20200616164816](https://user-images.githubusercontent.com/62371712/84753172-418d9b80-aff1-11ea-950b-57042ed61dc5.png)

**Comment by jennylove10** [comment]
> ![image](https://user-images.githubusercontent.com/28244886/84747804-57e42900-afea-11ea-80cd-74210e9bf4fc.png)
> 把这个only_output_key 设置为False试试？这么设置会只输出key而没有特征的

@tanmc123 改为only_output_key可以跑通啦,想再请教一下专家想把交集中具体传输密文要如何打印出来呐（打印的一直是这种object,有些我查看了源码是私有变量)
![QQ截图20200616164816](https://user-images.githubusercontent.com/62371712/84753172-418d9b80-aff1-11ea-950b-57042ed61dc5.png)
